environment/src/main/resources/application.yml

```diff
@@ -156,4 +156,17 @@ spring:
     maxActive: 30
 
 notification:
-  urls: http://localhost:3000/notifications
\ No newline at end of file
+  urls: http://localhost:3000/notifications
+
+xp:
+  path:
+    componentToReplace: "{crn}"
+    postfix: "/environment/{crn}"
+  protocol:
```

Comment:
why dont you set the https here?

Reply:
because it's https by default but I left it to be configurable

https://github.com/hortonworks/cloudbreak/pull/7244/files#diff-38bb82f2416cf83514e295898ba18688R39

Reply:
Yeh, I saw this constructor, but we try to handle the default values in applciation.yml instead of hardcoded values, as I know